src/fem/op_function_impl.hh

```diff
@@ -0,0 +1,294 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ *
+ * @file    op_function.hh
+ * @brief   Store finite element reinit functions.
+ * @author  David Flanderka
+ */
+
+#ifndef OP_FUNCTION_IMPL_HH_
```

Comment:
PAtchOp do samostatného souboru, na něm závisí PFeV a všechny "function".
Zrušit pro "function" oddělený impl soubor.